docs/widgets.md

```diff
@@ -238,17 +238,25 @@ By doing just this we were able to generate:
 
 ## Other Configuration
 
+#### Launch button text
```

Comment:
Suggestion: use headings semantically. The first heading under `##` should be `###`.